diode-server/reconciler/ingestion_processor.go

```diff
@@ -356,6 +376,24 @@ func normalizeIngestionLog(l []byte) []byte {
 	return re.ReplaceAll(l, []byte(`"ingestionTs":$1`))
 }
 
+func compressChangeSet(cs *changeset.ChangeSet) (string, error) {
+	csJSON, err := json.Marshal(cs)
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal changeset JSON: %v", err)
+	}
+
+	var brotliBuf bytes.Buffer
+	brotliWriter := brotli.NewWriter(&brotliBuf)
+	if _, err = brotliWriter.Write(csJSON); err != nil {
+		return "", fmt.Errorf("failed to compress changeset: %v", err)
+	}
+	if err = brotliWriter.Close(); err != nil {
+		return "", fmt.Errorf("failed to compress changeset: %v", err)
+	}
+
+	return base64.StdEncoding.EncodeToString(brotliBuf.Bytes()), nil
```

Comment:
Did we check the size of raw brotli bytes output vs brotli bytes encoded with base64?